requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-accelerate>=0.11.0
+accelerate>=0.11.0,!=0.20.*,!=0.21.0
```

Comment:
At this point, what do you think of going with `>=0.22` ?

Reply:
True, it's best to be on the latest version anyway. And since accelerate is not a strict dependency, users can still use skorch with different accelerate versions if they want.